doc/source/indexing.rst

```diff
@@ -1175,8 +1180,8 @@ If instead you don't want to or cannot name your index, you can use the name
 .. ipython:: python
    :suppress:
 
-   old_index = index
-   del index
+   old_index = index                                               # noqa: F821
+   del index                                                       # noqa: F821
```

Comment:
if this has `:suppress:` and it's not visible, and it's broken to need the `F821`, I guess we can simply get rid of it. Makes sense?